config/bin_version_strings.cfg

```diff
@@ -417,6 +417,7 @@ fsck.fat;;gplv3;"^fsck\.fat\ [0-9]+(\.[0-9]+)+?";"sed -r 's/fsck\.fat\ ([0-9]+(\
 fatlabel;;gplv3;"^fatlabel\ [0-9]+(\.[0-9]+)+?";"sed -r 's/fatlabel\ ([0-9]+(\.[0-9]+)+?).*/dosfstools:\1/'";
 mklost_plus_found;;unknown;"^mklost\+found\ [0-9]+(\.[0-9]+)+?\ \(.*\)$";"sed -r 's/mklost\+found\ ([0-9]+(\.[0-9]+)+?).*/e2fsprogs:\1/'";
 mkntfs;;gplv2;"mkntfs\ [0-9]+(\.[0-9]+)+?";"sed -r 's/mkntfs\ ([0-9]+(\.[0-9]+)+?).*/ntfsprogs:\1/'";
+mkntfs;;gplv2;"^mkntfs\ v[0-9]+(\.[0-9]+)+?";"sed -r 's/mkntfs\ ([0-9]+(\.[0-9]+)+?).*/ntfsprogs:\1/'";
```

Comment:
could we simplify this in the following way?
```
└─$ echo "mkntfs 1.2.3" | grep -E "mkntfs\ [0-9]+(\.[0-9]+)+?"
mkntfs 1.2.3
                                                                                                                                                                                                             
┌──(m1k3㉿emba)-[~/firmware-stuff/EMBAbite]
└─$ echo "mkntfs v1.2.3" | grep -E "mkntfs\ (v)?[0-9]+(\.[0-9]+)+?"
mkntfs v1.2.3
```
I think in your sed regex the "v" is missing